test/e2e/utils_test.go

```diff
@@ -399,3 +401,27 @@ func verifyCertificateRenewed(ctx context.Context, secretName, namespace string,
 		return true, nil
 	})
 }
+
+// create randomized string
+func randomStr(size int) string {
+	char := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+	rand.NewSource(time.Now().UnixNano())
+	var s bytes.Buffer
+	for i := 0; i < size; i++ {
+		s.WriteByte(char[rand.Int63()%int64(len(char))])
+	}
+	return s.String()
+}
+
+//replace string in bytes of ReadFile
```

Comment:
I remember "//xxx" without a space like "// xxx" could lead to the `ci/prow/code-check` job fail in QE private repo PRs.
```suggestion
// replace strings in bytes read from a file
```

Reply:
updated.